airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -1121,6 +1124,25 @@ def parse_response(self, response: requests.Response, stream_slice: Mapping[str,
         for record in response:
             yield record
 
+    def read_records(
+        self,
+        sync_mode: SyncMode,
+        cursor_field: List[str] = None,
+        stream_slice: Mapping[str, Any] = None,
+        stream_state: Mapping[str, Any] = None,
+    ) -> Iterable[Mapping[str, Any]]:
+        start_point = self.get_starting_point(stream_state=stream_state, stream_slice=stream_slice)
```

Comment:
you should add the PR description here as a comment to make it clear why this logic works for the next developer

Reply:
ok I will update